src/sentry/runner/commands/createuser.py

```diff
@@ -37,27 +37,39 @@ def _get_superuser():
     return click.confirm("Should this user be a superuser?", default=False)
 
 
-def _set_user_permissions(user):
-    from sentry.models import UserRole, UserRoleUser
-
-    if click.confirm(
+def _get_superadmin():
+    return click.confirm(
         "Should this user have Super Admin role? (This grants them all permissions available)",
         default=False,
-    ):
-        role = UserRole.objects.get(name="Super Admin")
-        UserRoleUser.objects.create(user=user, role=role)
+    )
+
+
+def _get_staff():
+    return click.confirm("Should this user be staff?", default=False)
+
+
+def _set_superadmin(user):
+    from sentry.models import UserRole, UserRoleUser
+
+    role = UserRole.objects.get(name="Super Admin")
+    UserRoleUser.objects.create(user=user, role=role)
 
 
 @click.command()
 @click.option("--email")
 @click.option("--password")
 @click.option("--superuser/--no-superuser", default=None, is_flag=True)
+@click.option("--superadmin/--no-superadmin", default=False, is_flag=True)
+@click.option("--staff/--no-staff", default=None, is_flag=True)
 @click.option("--no-password", default=False, is_flag=True)
 @click.option("--no-input", default=False, is_flag=True)
 @click.option("--force-update", default=False, is_flag=True)
 @configuration
-def createuser(email, password, superuser, no_password, no_input, force_update):
+def createuser(email, password, superuser, superadmin, staff, no_password, no_input, force_update):
```

Comment:
What's the permissions hierarchy between superuser, superadmin, staff? I suggest consistently ordering them from least to greatest permissions (or reverse, but consistently).